### PR TITLE
Cent os gpu enabled

### PR DESCRIPTION
**Pull request by yiakwy**
1. fix installation bugs for gpu driver conflict. see ["hwo_to_prepare_centos_gpu_enabled_image_for_ROS_perception_module.md"](https://github.com/yiakwy/apollo/blob/CentOS_gpu_enabled/docs/howto/how_to_prepare_centosbased_gpu_enabled_image_for_ROS_perception_module.md)

2. fixed previous baidu docs typos

3. add ["how_to_understand_architecture_and_workflow"](https://github.com/yiakwy/apollo/blob/CentOS_gpu_enabled/docs/howto/how_to_understand_architecture_and_workflow.md)

### CODE REVIEW

**Comment by yiakwy** [comment]
Hello Please review

File: modules/common/data/global_flagfile.txt [hunk 1]
```diff
@@ -8,3 +8,7 @@
 --vehicle_config_path=modules/common/data/mkz_config.pb.txt
 
 --map_dir=modules/map/data/sunnyvale_loop
+
+--map_dir=modules/map/data/sunnyvale_loop
```
Comment (ycool) [inline modules/common/data/global_flagfile.txt]: remove this file change.

File: modules/drivers/velodyne/README_cn.md [hunk 2]
```diff
@@ -24,8 +24,8 @@ velodyne驱动是以nodelet的形式实现的，包含了:
 
 ```bash 
 # in dev docker
-cd /apolloe
-bash apollo.sh build_velodyne
+cd /apollo
+bash apollo.sh buildvelodyne
```
Comment (ycool) [inline modules/drivers/velodyne/README_cn.md]: It is build_velodyne
https://github.com/ApolloAuto/apollo/blob/master/apollo.sh#L650

File: modules/planning/tasks/qp_spline_path/qp_spline_path_generator.cc [hunk 3]
```diff
@@ -296,7 +296,6 @@ bool QpSplinePathGenerator::AddConstraint(const QpFrenetFrame& qp_frenet_frame,
   if (init_trajectory_point_.v() > qp_spline_path_config_.uturn_speed_limit()) {
     spline_constraint->AddPointSecondDerivativeConstraintInRange(
         init_frenet_point_.s(), init_frenet_point_.ddl(), kBoundaryEpsilon);
-  }
```
Comment (ycool) [inline modules/planning/tasks/qp_spline_path/qp_spline_path_generator.cc]: why change planning code when adding centos support?

File: modules/planning/tasks/qp_spline_path/qp_spline_path_generator.cc [hunk 4]
```diff
@@ -321,7 +320,6 @@ bool QpSplinePathGenerator::AddConstraint(const QpFrenetFrame& qp_frenet_frame,
   if (!is_change_lane_path_) {
     spline_constraint->AddPointDerivativeConstraintInRange(
         evaluated_s_.back(), 0.0, kEndPointBoundaryEpsilon);
-  }
```
Comment (Capri2014) [inline modules/planning/tasks/qp_spline_path/qp_spline_path_generator.cc]: Also this one, seems unbalanced bracket?

**Comment by Capri2014** [comment]
Nice work! @yiakwy

**Comment by Capri2014** [comment]
Hi @yiakwy  could you please do a git rebase to upstream master before push?

File: modules/common/data/global_flagfile.txt (hunk 1 above)
Comment (yiakwy) [reply modules/common/data/global_flagfile.txt]: Thank you, fixed already.

File: modules/planning/tasks/qp_spline_path/qp_spline_path_generator.cc (hunk 4 above)
Comment (yiakwy) [reply modules/planning/tasks/qp_spline_path/qp_spline_path_generator.cc]: Thank you, fixed already.

**Comment by yiakwy** [comment]
@Capri2014 I have rebased the branch on origin master latest commit '384c3aa370a637d2ef970f64d4a7bb293615fa05'.  Hope it works!

**Comment by yiakwy** [comment]
@ycool, @Capri2014  Hi, I have fixed the errors and rebase the branch onto the origin master latest commit "384c3aa370a637d2ef970f64d4a7bb293615fa05". Any other suggestions to help me to make it work? Thank you!

**Comment by lianglia-apollo** [comment]
The lint check is failed. Could you verify using "bash apollo.sh check"?

**Comment by ycool** [comment]
Thanks a lot for your contribution. I am going to merge your CL and we will fix the lint problem. And next time you need run `bash apollo.sh check` before pushing. Thanks.

**Pull request merged**